packages/sheet/src/index.js

```diff
@@ -84,20 +84,27 @@ export class StyleSheet {
     this.before = null
   }
 
+  _insertTag = (tag = createStyleElement(this)) => {
```

Comment:
This will probably be slightly smaller if we don't do a default arg and instead call it with that where we need to

Reply:
Agreed - proposed change is much better.